src/tslint/overrideCallsSuperRule.ts

```diff
@@ -34,8 +33,7 @@ class OverrideCallsSuperWalker extends RuleWalker {
 
     visitMethodDeclaration(node: ts.MethodDeclaration): void {
         const methodName = node.name.getText();
-
-        if (_.includes(this._methodNamesToCheck, methodName)) {
+        if (this._methodNamesToCheck.indexOf(methodName) >= 0) {
```

Comment:
Use `.includes` or shim, similar to `.find`.
```ts
export function includes(array: T[], value: T): boolean {
    if (isFunction(array.includes) {
        return array.includes(value);
    }
    return array.indexOf(value) !== -1;
}
```

Reply:
I think we don't need to add a new function for `tslint` rules., I think in the Node.JS environment is ok to use `.includes`.